api/src/test/java/io/opentelemetry/api/OpenTelemetryTest.java

```diff
@@ -59,11 +60,13 @@ void after() {
 
   @Test
   void testDefault() {
-    assertThat(OpenTelemetry.getGlobalTracerProvider()).isInstanceOf(DefaultTracerProvider.class);
-    assertThat(OpenTelemetry.getGlobalTracerProvider())
+    Assertions.assertThat(OpenTelemetry.getGlobalTracerProvider())
```

Comment:
odd that this is not being statically imported any more. I won't hold up the PR for it, since it's minor, but odd that it snuck in here.

Reply:
haha :) this is because I disabled imports of inner classes/methods. Do you want to allow imports for the assertions I expect.

Reply:
Did the refactoring myself, this change did not happen. so it needs to be reverted

Reply:
Thanks for finding! I'm used to IntelliJ refactoring causing havoc with static imports, just one file being hit was relatively tame